jobQueue/jobWrapper.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * /jobQueue/jobWrapper.js
+ *
+ * Expsoses a wrapper to create jobs and save them to the redis queue for
+ * it to be processed by the workers later. The "kue" library defined in the
+ * setup is used for this purpose.
+ */
+
+'use strict'; // eslint-disable-line strict
+
+const jobQueue = require('./setup').jobQueue;
+
+const delayToRemoveJobs = 3000;
```

Comment:
Why hard-coded here? If this delay value of 3000 is the *only* value which could ever work, then add a comment explaining why it has to be that way. Otherwise, move to config json (and document in readme).